hack/verify-gendocs.sh

```diff
@@ -104,8 +73,10 @@ set -e
 rm -rf "${TMP_DOCROOT}"
 if [ $ret -eq 0 ]
 then
-	echo "${DOCROOT} up to date."
+  echo "${DOCROOT} up to date."
 else
-	echo "${DOCROOT} is out of date. Please run hack/run-gendocs.sh"
-	exit 1
+  echo "${DOCROOT} is out of date. Please run hack/run-gendocs.sh"
```

Comment:
But it does cause it to be removed, because of:

https://github.com/GoogleCloudPlatform/kubernetes/pull/6765/files#diff-ef2873ebed657da627ed30f61dea2964R117